.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   },
   plugins: ['react', '@typescript-eslint'],
   rules: {
+    'react/no-unknown-property': ['error', { ignore: ['css'] }],
```

Comment:
why did have to you turn this off?

Reply:
I tried to fix the error in 'CSS' props in HTML tags like
`<div **css**={css``}>/div>`
THe bolding CSS makes not known property error. After ignoring codes like this, the error disappeared but still have a problem, so I didn't use the codes. I will delete it.